### PR TITLE
18082 Condensed Audit Logs into one page

### PR DESCRIPTION
**Pull request by spark-c**
This PR combines the separate Audit Logs - related subpages back into a single document, along with some fixes to satisfy linter and grammarly.

### CODE REVIEW

**Comment by spark-c** [comment]
Getting an error on build that I didn't catch initially:

```
[ERROR] Error: Invalid sidebar file at "sidebars.js".
These sidebar document ids do not exist:
- advanced-concepts/audit-logs/README
- advanced-concepts/audit-logs/filtering-audit-logs
- advanced-concepts/audit-logs/log-contents
- advanced-concepts/audit-logs/tracked-events
```

Looks like docusaurus (via sidebars.js) is still expecting to find the pages that I've removed, and the build is failing as a result. Do I need to manually update this file when I add or remove pages? Not sure if I'm just missing a step somewhere.

**Comment by jnikhila** [comment]
> Getting an error on build that I didn't catch initially:
> 
> ```
> [ERROR] Error: Invalid sidebar file at "sidebars.js".
> These sidebar document ids do not exist:
> - advanced-concepts/audit-logs/README
> - advanced-concepts/audit-logs/filtering-audit-logs
> - advanced-concepts/audit-logs/log-contents
> - advanced-concepts/audit-logs/tracked-events
> ```
> 
> Looks like docusaurus (via sidebars.js) is still expecting to find the pages that I've removed, and the build is failing as a result. Do I need to manually update this file when I add or remove pages? Not sure if I'm just missing a step somewhere.

Yes, you'll have to manually update the ```sidebar.js``` and make the necessary changes for the audit log.

**Review by jnikhila — COMMENTED** [review]
Please add redirect rules for the files that are removed to ```vercel.json``` file.

File: website/docs/advanced-concepts/audit-logs.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    - Deleting apps
+
+- Changes to Administrator settings
+    - Instance configuration changes
+
+- User sign-up and login activity
+
+For an exhaustive list of events, see [Tracked Events](#tracked-events).
+
+## Filtering audit logs
```
Comment (danciaclara) [inline website/docs/advanced-concepts/audit-logs.md:34]: Let's move this below Log Contents

**Review by jnikhila — APPROVED** [review]

**Comment by spark-c** [comment]
Deployment successful, has resolved #543

**Pull request merged**